README.md

```diff
@@ -324,10 +324,14 @@ NVM_IOJS_ORG_MIRROR=https://iojs.org/dist nvm install iojs-v1.0.3
 
 ### .nvmrc
 
-You can create a `.nvmrc` file containing version number in the project root directory (or any parent directory).
-`nvm use`, `nvm install`, `nvm exec`, `nvm run`, and `nvm which` will all respect an `.nvmrc` file when a version is not supplied on the command line.
+You can create a `.nvmrc` file containing node version number in the
+project root directory (or any parent directory).
```

Comment:
please don't hard-wrap sentences; everything anybody uses to view text in the last decade or two has soft-wrapping :-)

Reply:
Except github diffs, terminals and text/plain rendering by browsers, to name the first three things that came to mind.

Reply:
Terminals all can soft-wrap, as do github diffs (they omit the line number on soft-wrapped lines).